experimental/web_dashboard/lib/src/utils/day_helpers.dart

```diff
@@ -0,0 +1,11 @@
+extension DayUtils on DateTime {
```

Comment:
nit: copyright header.

Reply:
I'm chuffed to see a good use case for extension methods in one of our samples. 😄

Reply:
Also, I've been watching so much of the Great British Bake-Off during quarantine that apparently I now say "chuffed."

Reply:
good bake, err, catch.